core/unit_test/TestTeam.hpp

```diff
@@ -849,6 +849,13 @@ struct TestScratchTeam {
         Functor(), result_type(&error_count));
     Kokkos::fence();
     ASSERT_EQ(error_count, 0);
+
+    Kokkos::parallel_reduce(
+        team_exec.set_scratch_size(1, Kokkos::PerTeam(team_scratch_size),
+                                   Kokkos::PerThread(thread_scratch_size)),
+        Functor(), Kokkos::Sum<typename Functor::value_type>(error_count));
+    Kokkos::fence();
+    ASSERT_EQ(error_count, 0);
```

Comment:
...or this should be in this pull request.

Reply:
For all cases of files that shouldn't be in this PR, I've checked out the `develop` versions of these files, and there doesn't seem to be any difference between the versions.  I will look back through the commits to see how these errant files ended up in this PR.